apps/reactotron-app/src/renderer/models/Connection.ts

```diff
@@ -9,7 +9,7 @@ export const ConnectionModel = t
 
     platform: t.enumeration(["ios", "android", "browser"] as const),
     name: "",
-    platformVersion: "",
+    platformVersion: t.optional(t.union(t.number, t.string), ""),
```

Comment:
@jamonholmgren - `platformVersion` is coming back as a number on Android, and a string on iOS. This handles both and defaults to an empty string.